spring-native-aws-lambda-infra/src/main/java/com/coffeebeans/springnativeawslambda/infra/SpringNativeAwsLambdaStack.java

```diff
@@ -57,7 +57,7 @@ public SpringNativeAwsLambdaStack(@NotNull final Construct scope,
 
     final Queue successQueue = createFifoQueue(SQS_SUCCESS_QUEUE_ID, true, MESSAGE_GROUP);
     final SqsSubscription successQueueSubscription = createSqsSubscription(successQueue);
-    final Topic successTopic = createFifoTopic(SNS_SUCCESS_TOPIC_ID, true, true);
```

Comment:
AWS does not allow fifo topics as lambda destination anymore (or at least fail when provided with fifo as destination for lambda)